algosdk/abi/base_type.py

```diff
@@ -45,7 +44,7 @@ def encode(self, value):
         pass
 
     @abstractmethod
-    def decode(self, value_string):
+    def decode(self, bytestring):
```

Comment:
Sometimes we are using type hints, and sometimes we aren't. What is our roadmap in this regards, or is just ad-hoc at the moment?

Reply:
Good observation, I think PyTeal uses more type hints (it also has `mypy` checks), but I don't think our SDK does it right now. I think it would be nice for the future though.